ci/docker/min.build.Dockerfile

```diff
@@ -26,7 +26,7 @@ MINIMAL_INSTALL=1 PYTHON=${PYTHON_VERSION} ci/env/install-dependencies.sh
 rm -rf python/ray/thirdparty_files
 
 # install test requirements
-python -m pip install -U pytest==7.0.1 pip-tools==7.3.0
+python -m pip install -U pytest==7.0.1 pytest-asyncio==0.16.0 pip-tools==7.3.0
```

Comment:
this is to fix the aiohttp issue?

Reply:
not really. this is that there are several tests in the test suite that is async, and they are skipped. I was trying to understand if it has anything to do with the event loop thing.

will revert this.

Reply:
we probably should add this.. but I can do it in another PR.